### PR TITLE
Use PostgresVersion for postgres images

### PR DESCRIPTION
**Pull request by annymsMthd**
Signed-off-by: Joshua Benjamin <joshuabnjmn@gmail.com>

### CODE REVIEW

**Comment by tamalsaha** [comment]
> Will pkg/docker/images.go be needed anymore after this?

No. We can remove that.

> Also how would we go about running the e2e tests for this?

I don't think we need to do anything special. Just adapt the existing tests for `PostgresVersion` crd.

**Comment by annymsMthd** [comment]
@tamalsaha let me know if you see any other issues with this PR

**Comment by tamalsaha** [comment]
/LGTM .

@the-redback , can you review this, too?

**Comment by the-redback** [comment]
Also, Revendor. These will be enough. Other things work fine. :)

**Comment by annymsMthd** [comment]
I'm still having a few issues when revendoring. When I use glide it wants to update a lot of things vs just the kubedb stuff.

**Comment by tamalsaha** [comment]
Not sure that is the deal with glide, but I have gone ahead and revendored apimachinery in #163 . So, just rebase on master.

File: test/e2e/framework/admission_controller.go [hunk 1]
```diff
@@ -78,8 +78,6 @@ func (f *Framework) RunOperatorAndServer(kubeconfigPath string, stopCh <-chan st
 	serverOpt.RecommendedOptions.Authorization.RemoteKubeConfigFile = kubeconfigPath
 	serverOpt.RecommendedOptions.Authentication.RemoteKubeConfigFile = kubeconfigPath
 
-	serverOpt.ExtraOptions.Docker.Registry = DockerRegistry
```
Comment (annymsMthd) [inline test/e2e/framework/admission_controller.go]: Should this part be readded?

**Pull request merged**